tox.ini

```
@@ -115,7 +113,8 @@ usedevelop = false
 deps = pre-commit>=1.17.0
 commands=
     python -m pre_commit run --color=always {posargs:--all}
-extras =
+setenv =
+    PIP_CONSTRAINT=/dev/null
```

when creating/recreating the tox env this doesn't play nice with Windows, I had to instead get it working with an empty variable using:

```suggestion
    PIP_CONSTRAINT=
```